pyecharts/charts/basic_charts/geo.py

```diff
@@ -53,8 +53,9 @@ def add(
         self._zlevel += 1
 
         data = []
-        if type_ == ChartType.LINES and self._coordinate_system == "bmap":
-            data = data_pair
+        if self._coordinate_system == "bmap":

```

Comment:
GeoChartBase shall not take care of the details of child. Otherwise, all children come back to base for help.